src/assets/styles/50-utilities/_scaffolding.scss

```diff
@@ -36,5 +36,31 @@
 
   @include mq(md) {
     @include grid-column(1, 13);
+    position: relative;
+  }
+}
+
+
+// "wide" content has to appear above sticky content
+// such as the in-page nav
+.u-scaffold-grid-item--wide > * {
+  @include mq(md) {
+    position: relative;
+    z-index: var(--z-medium);
+  }
+}
+
+// linear gradient applied to give a fading out appearance to sticky content layered behind it
+.u-scaffold-grid-item--wide:after {
+  @include mq(md) {
+    background: linear-gradient(180deg, var(--hack-safari-transparent-white) 0, var(--colour-white) 10%, var(--colour-white) 90%, var(--hack-safari-transparent-white) 100%);
+    bottom: -100px;
+    content: '';
+    left: 0;
+    pointer-events: none;
+    position: absolute;
+    right: 0;
+    top: -100px;
+    z-index: var(--z-low);
   }
 }
```

Comment:
I feel like we may benefit from these styles being tied to their own specific selector, rather than the `.u-scaffold-grid-item`. I think it makes sense from a readabilty standpoint as what we're doing here feels like a workaround.

What do you think?

Reply:
Makes sense. I've created a utility class and also a mixin. The latter I have included used directly within this file to keep complexity within the [scaffold mapping](https://github.com/wellcometrust/corporate-react/blob/690e8e98aa43da6b7c8eac503d99f490862d8dd1/src/utils/scaffold-mapping.tsx#L153) to minimum.